test/test_underlying_mdp.jl

```diff
@@ -11,7 +11,7 @@ let
 
     solver = ValueIterationSolver(max_iterations = 100)
     mdp_policy = solve(solver, mdp)
-    pomdp_policy = solve(solver, pomdp)
+    pomdp_policy = solve(solver, UnderlyingMDP(pomdp))

```

Comment:
This test does not make sense anymore

Reply:
Yeah, but it doesn't hurt anything. At least it exercises a lot of the functions. I think it is fine